cardano-config/src/Cardano/TracingOrphanInstances/Byron.hs

```diff
@@ -61,6 +66,53 @@ instance ToObject ApplyMempoolPayloadErr where
       , "error" .= String (show iFaceErrr)
       ]
 
+instance ToObject ByronLedgerUpdate where
+  toObject verb (ByronUpdatedProtocolUpdates protocolUpdates) =
+    mkObject $
+      [ "kind"            .= String "ByronUpdatedProtocolUpdates"
+      , "protocolUpdates" .= map (toObject verb) protocolUpdates
+      ]
+
+instance ToObject ProtocolUpdate where
+  toObject verb (ProtocolUpdate updateVersion updateState) =
+    mkObject $
+      [ "protocolUpdateVersion" .= updateVersion
+      , "protocolUpdateState"   .= toObject verb updateState
+      ]
```

Comment:
Are we purposefully not providing a `kind` field here?

Reply:
Probably not. I suggest add it.

Reply:
Oh, not purposefully, will fix it. BTW, if a `kind` field is mandatory, maybe `mkObject` should take it as an argument?